app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -765,9 +765,10 @@ public String revisionsByFilename(String filename) throws IOException {
                     .param("gsroffset",offset)
                     .param("gsrsearch", query)
                     .param("prop", "imageinfo")
+                    .param("iiprop", "url|extmetadata")
                     .get();
-            imageNodes= customApiResult.getNodes("/api/query/pages/page/@title");
-            authorNodes= customApiResult.getNodes("/api/query/pages/page/imageinfo/ii/@user");
+            imageNodes= apiResult.getNodes("/api/query/pages/page/@title");
```

Comment:
`imageNodes` and `authorNodes` is no longer used. You can remove it.

Reply:
ok I'll do that :+1: